packages/sources/bitgo-reserves/src/endpoint/reserves.ts

```diff
@@ -0,0 +1,19 @@
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
```

Comment:
We can use PoRProviderResponse and PoRProviderEndpoint here

Reply:
Debatable if there's any value. The advantage of PoRProviderResponse is to force returning a `ripcord` field, which this API+endpoint does/will not have (as far as we know at this point).

Given that we'd just end up forcing ripcord=false, what do you think?

Reply:
I am fine with both

Reply:
alright, i'm leaning towards without PoRProviderResponse. Happy to revisit if the API changes and they decide to add a ripcord in the future.